src/PowerTools/source.extension.vsixmanifest

```diff
@@ -1,9 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Vsix xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Version="1.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2010">
-  <Identifier Id="2b119c79-9836-46e2-b5ed-eb766cebbf7c">
-    <Name>Entity Framework Power Tools Beta 5</Name>
-    <Author>Microsoft</Author>
-    <Version>0.9.0.0</Version>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
```

Comment:
@lajones  Yes, it is correct, and trust me it works  (as demonstrated) - there is no 3.0.0 - the main point (as stated earlier) is that the VSIX 3 file contains two additional json files (provided by using the Dev15 build nugte package), and that the manifest specifies prerequisites... You can look inside the VSIX to see what I mean.

@divega @bricelam  You need the build nuget package (which injects the two additionla json files) , and you must specify at least one prerequisite